### PR TITLE
Example linear trend

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: src/models/locallineartrend.jl [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+LocalLinearTrend model
+```
+
+See more on [Finland road traffic fatalities](@ref)
+
+# References
+ * Durbin, James, & Siem Jan Koopman. (2012). "Time Series Analysis by State Space Methods: 
+    Second Edition." Oxford University Press. pp. 44
+"""
+mutable struct LocalLinearTrend <: StateSpaceModel
```
Comment (raphaelsaavedra) [inline src/models/locallineartrend.jl:27]: Why not just `LinearTrend`?

File: docs/src/examples.md [hunk 2]
```diff
@@ -79,3 +82,45 @@ smoother_output = kalman_smoother(model)
 plot!(plt, nile.year, filtered_estimates(filter_output), label = "Filtered level")
 plot!(plt, nile.year, smoothed_estimates(smoother_output), label = "Smoothed level")
 ```
+
+# Log of airline passengers
```
Comment (raphaelsaavedra) [inline docs/src/examples.md:86]:
```suggestion
## Log of airline passengers

TODO
```

File: docs/src/examples.md [hunk 3]
```diff
@@ -79,3 +82,45 @@ smoother_output = kalman_smoother(model)
...
+
+# Finland road traffic fatalities
```
Comment (raphaelsaavedra) [inline docs/src/examples.md:88]:
```suggestion
## Finland road traffic fatalities
```

File: docs/src/manual.md [hunk 4]
```diff
@@ -7,10 +7,7 @@ The package provides a variaty of pre-defined models, fell free to contribute if
 ### Local Level
```
Comment (raphaelsaavedra) [inline docs/src/manual.md:7]: Change to this since they're all together?

```suggestion
### Models
```

**Review by raphaelsaavedra — CHANGES_REQUESTED** [review]

File: src/models/locallineartrend.jl (hunk 1 above)
Comment (guilhermebodin) [reply src/models/locallineartrend.jl:27]: They call it local linear trend on the book

Captura de Tela 2020-09-05 às 18 36 21

Comment (guilhermebodin) [reply src/models/locallineartrend.jl:27]: They call it local linear trend on the book

Captura de Tela 2020-09-07 às 16 25 46

**Pull request merged**